host/tests/L3_TestCases/dsFPD/dsFPDHelperClass.py

```diff
@@ -53,16 +53,17 @@ def __init__(self, testName:str, qcId:str, log:logModule=None ):
 
         # Load test setup configuration
         self.testSetup = ConfigRead(self.testSetupPath, self.moduleName)
-        #open Hal Seesion
+        #open Hal Session
         self.hal_session = self.dut.getConsoleSession("ssh_hal_test")
+        self.deviceTestSetup = self.cpe.get("test")
 
          # Create user response Class
         self.testUserResponse = utUserResponse()
 
         # Get path to device profile file
-        self.deviceProfile = os.path.join(dir_path, self.cpe.get("test").get("profile"))
+        self.moduleConfigProfileFile = os.path.join(dir_path, self.deviceTestSetup.get("profile"))
 
-        self.deviceDownloadPath = self.cpe.get("target_directory")
+        self.targetPath = self.cpe.get("target_directory")
```

Comment:
Follow a similar structure like in dsAudio for the target path

Reply:
Updated tagrgetPath to include a folder with module name.
This will create an extra folder dsFPD in the target Path provided in the configuration file.